src/main/webapp/feed.html

```diff
@@ -33,18 +42,116 @@ <h1>
       import { app } from "/scripts/init.js"
 
       let recipeList = document.getElementById("recipe-list")
+      let tagSelect = document.getElementById("tag-select")
+
+      window.addFilter = addFilter
+
+      window.onload = function () {
+        // Fetch all recipes to populate our list
+        fetch("/api/post").then(async (response) => {
+          if (!response.ok) {
+            alert("Error " + response.status.toString())
+          } else {
+            let recipes = await response.json()
+            recipes.forEach((recipe) => {
+              recipeList.innerHTML +=
+                '<li><a href="recipe-display.html?recipeID=' + recipe.id + '">' + recipe.title + "</a></li>"
+            })
+          }
+        })
+
+        // Fetch all recipe tags to populate our select form
+        fetch("/api/tag").then(async (response) => {
+          if (!response.ok) {
+            alert("Error " + response.status.toString())
+          } else {
+            let tags = await response.json()
+            tags.forEach((tag) => {
+              tagSelect.innerHTML += '<option value="' + tag.id + '">' + tag.name + "</option>"
+            })
+          }
+        })
+      }
+
+      // Performs retrieval of recipes based on the filters currently contained in <div> 'selected-filters'
+      function getRecipes() {
+        const tagIDs = getCurrentFilters()
+        let recipeList = document.getElementById("recipe-list")
+        let queryString = "/api/post"
+
+        if (tagIDs.length > 0) {
+            queryString += "?tagIDs=" + tagIDs[0]
+            for (var x = 1; x < tagIDs.length; x++) {
+                queryString += "&tagIDs=" + tagIDs[x];
```

Comment:
I think you can use join() here with &tagIDs= as the separator.

Reply:
@bradleypickard  i might need a little help on this one

Reply:
```
qsTags = tagIDs.map(tag => "tagIds=" + tag)
queryString += "?" + qsTags.join("&")
```
think that should do it